rollup.config.js

```diff
@@ -140,17 +140,17 @@ const makePlugins = (isProduction = false) => [
   isProduction ? terserMinified : terserPretty
 ].filter(Boolean);
 
-const config = {
-  input: './src/index.ts',
+const makeConfig = (input) => ({
+  input,
   external: externalTest,
   treeshake: {
     propertyReadSideEffects: false
   }
-};
+});
 
 export default [
   {
-    ...config,
+    ...makeConfig('./src/index.ts', './dist'),
```

Comment:
Rollup has a feature to alias separate inputs to names that generate separate bundles that import one another as necessary.

This is good because it means that people can use both the core entry and the main-entry without duplicating code or running into global state issues

Reply:
But can you generate them at separate places then?